Atomic/syscontainers.py

```diff
@@ -583,6 +576,20 @@ def _amend_values(self, values, manifest, name, image, image_id, destination, pr
         return values
 
 
+    def _canonicalize_location(self, destination):
+        # Under Atomic, get the real deployment location if we're using the
+        # system repo. It is needed to create the hard links.
+        if self.get_ostree_repo_location() != '/ostree/repo':
+            return destination
+        try:
+            sysroot = OSTree.Sysroot()
+            sysroot.load()
+            osname = sysroot.get_booted_deployment().get_osname()
+            destination = os.path.realpath(os.path.join("/ostree/deploy/", osname, os.path.relpath(destination, "/")))
+        except: #pylint: disable=bare-except
+            pass
```

Comment:
nit: Seems like there should be a log line here (unless having ``OSTree.Sysroot()``or ``get_booted_deployment().get_osname()`` raising are not "important" errors.